src/components/structures/MatrixChat.js

```diff
@@ -140,6 +141,7 @@ module.exports = React.createClass({
 
     componentWillMount: function() {
         this.favicon = new Favico({animation: 'none'});
+        this.guestCreds = null;
```

Comment:
This is an old-style React component so this is the constructor, but I can comment it.